packages/ui/README.md

```diff
@@ -37,6 +54,9 @@ Once the dependencies have run follow these login instruction:
 
 ## Development
 
+You need to have built `bichard7-next-core/packages/core` and `bichard7-next-core/packages/common` before you can run
+this package.
```

Comment:
Maybe link to your new "Running Packages locally" section in the root README?

Reply:
I think I'll link to this.